automation/test.sh

```diff
@@ -38,7 +38,7 @@ readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluste
 # All the other lanes will fail immediately, saving CI resources,
 # but still guarding from getting the PR merged, until the line
 # is restored to an empty string.
-export KUBEVIRT_LANE_FOCUS=""
+export KUBEVIRT_LANE_FOCUS="kind-k8s-sriov-1.17.0"
```

Comment:
Could you please remove this focus? It will be removed as part of #3832 anyway and cause merge conflicts and it introduces the mentioned issues there.

Reply:
You can make your PR a draft instead and explicitly trigger the lanes you care about.

Reply:
Thanks for the heads up